src/services/videoconference/index.js

```diff
@@ -0,0 +1,135 @@
+const {
+	BadRequest,
+	Forbidden,
+	NotFound,
+	GeneralError,
+	NotImplemented,
+	FeathersError,
+} = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)